Tests/SonarQube.TeamBuild.PostProcessor.Tests/Infrastructure/MockTeamBuildSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * SonarQube Scanner for MSBuild
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.Integration.Interfaces;
+
+namespace SonarQube.TeamBuild.PostProcessor.Tests
+{
+    internal class MockTeamBuildSettings : ITeamBuildSettings
```

Comment:
The _ITeamBuildSettings_ interface only has getters so _Mock_ can't be used to set the results.
This is the simplest workaround that doesn't involve changing the product code.